tests/core/test_can_validate.py

```diff
@@ -19,7 +19,42 @@ def test_inspect():
     assert result.validated == ["TCF7"]
 
     # clean up
-    bt.Gene.filter().all().delete()
+    bt.Gene.filter().delete()
+
+
+# if a record was added to the DB via a different source
+# it will still be validated because it's in the DB
+# we might want to introduce a strict mode in the future
```

Comment:
The comment here is confusing because you're introducing a strict mode in this PR and even testing it in this test?

Can you please prettify the code in this test, e.g. by not repeating `bt.Source.get(name="2dfU")` 6 times?